cmd/popp/popp.go

```diff
@@ -158,6 +158,12 @@ func run() error {
 	}
 	fmt.Printf("==> Started pop exchange\n")
 
+	// remove unwanted blocks that might be in the blockstore but are removed from the index
+	err = exch.Index().CleanBlockStore(ctx)
```

Comment:
Should we run this in a goroutine so we can start the bootstrapping process in parallel? I imagine it could take a while for a large blockstore.

Reply:
Sure that should work as well. Note that this is why the blockstore has a lock...